contributing.md

```diff
@@ -1,14 +1,173 @@
-#### How to reach us
-The mailing list for developers : systemroles@lists.fedorahosted.org
+# Contributing to the Network Linux System Role
 
-[Subscribe to the mailing list here](https://lists.fedorahosted.org/admin/lists/systemroles.lists.fedorahosted.org/)
+## Where to start
 
-[Archive of the mailing list](https://lists.fedorahosted.org/archives/list/systemroles@lists.fedorahosted.org/)
+- **Bugs and needed implementations** are listed on [Github
+  Issues](https://github.com/linux-system-roles/network/issus). Issues labeled
+with [**help wanted**](https://github.com/linux-system-roles/network/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) are likely to be suitable for new contributors!
```

Comment:
This line seems to be longer than 88 characters.

Reply:
Yes! The problem is that the link gets altered and stops working when I break it in two lines